README.md

```diff
@@ -99,13 +101,24 @@ You should get a HTTP 200 response code and see the notification in either Slack
 1. Run these commands after adjusting the repo, image name, and tag in this example
     ```
     # build image
-    docker build -t robjahn/notification-service:0.1.0 .
+    docker build -t keptnexamples/notification-service:0.2.0 .
```

Comment:
Can we use a differ org name? Something like "keptn-contrib" or "keptn-extensions".

Reply:
"keptncontrib" makes sense, then it's in line with the GitHub organization names. Last time I checked, hyphens were not allowed in docker hub organizations.

Reply:
@dirkwall - happy to use a new "keptncontrib" org.  Just let me know when available and I will adjust the travis scripts.

Reply:
I worked with Dirk and have adjusted to push images to this new org. 
  https://cloud.docker.com/u/keptncontrib/repository/docker/keptncontrib/notification-service